db/migrate/20230222101705_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index:true
+      t.references :post, null: false, foreign_key: true, index:true
```

Comment:
- [ ] This is throwing an error because you are using post as reference before creating a post migration, so the comment migration should happens after the post one
![image](https://user-images.githubusercontent.com/62705947/220732200-4fbbcdf8-7985-4514-b0a2-de82aff29c26.png)

Reply:
Hi @GzimAsani. This problem is resolved when I add the foreign key relations later i.e. AddPostToComments and AddPostToLikes, So would you recommend that I delete and reorder the migration files instead?

Reply:
Hi @keebot-png Well, I cannot migrate your database because it is throwing this error and all the other migrations after this are cancelled, so in order for a user to run your migrations you need to fix this problem

Reply:
No worries @GzimAsani it was easy to restructure. I sorted it out💪